src/styles/partials/components/_card-container.scss

```diff
@@ -0,0 +1,42 @@
+.dg_card {
+  border: $border-width-small solid $gray;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: space-between;
+  position: relative;
+  margin-bottom: $margin-large;
+  padding: $padding-large;
+  width: 100%;
+
+  .dg_card-content-center {
+    display: block;
+    text-align: center;
+
+    .dg_icon-container {
```

Comment:
I don't think you need to dump this in here, I think you have it covered on line 36. It wouldn't be specific to centered content

Reply:
Updated and committed.